db/migrate/20250507235501_convert_ite_client_id_to_int.rb

```
@@ -0,0 +1,67 @@
+          unique: idx.unique
+        }
+
+        add_index table, idx.columns, **options
+      end
+    end
+    
+    # add foreign key constraints back to foreign key tables
+    foreign_key_tables.each do |table|
+      add_foreign_key table, :ite_client_records, column: :ite_client_id, primary_key: :ite_client_id
+    end
+  end
```

RuboCop (1.75.5)

[convention] Cyclomatic complexity for `change` is too high. [12/10]

(Metrics/CyclomaticComplexity)